src/app/example_configs/plugin-logging.yaml

```diff
@@ -1,30 +1,30 @@
-level: INFO
+version: 1
+disable_existing_loggers: False
+
 handlers:
   stdout:
     class: "logging.StreamHandler"
-    formatter: stdout
-    level: INFO
-    stream: "ext://sys.stdout"
-  file:
-    backupCount: 20
-    class: "logging.handlers.RotatingFileHandler"
-    encoding: utf8
     formatter: default
-    level: INFO
-    maxBytes: 10485760
-    filename: "plugin.log"
+    stream: "ext://sys.stdout"
+
+#  Uncomment this and the handler in the root->handlers section to enable file-based
+#  plugin logging
+#  file:
+#    backupCount: 5
+#    class: "logging.handlers.RotatingFileHandler"
+#    encoding: utf8
+#    formatter: default
+#    level: INFO
+#    maxBytes: 10485760
+#    filename: "logs/$instance_name.log"
+
```

Comment:
Just wanted to make that we document this. The filename is in relation to where the plugin is executed. So just kicked off a remote plugin and it built it in the directory I was running in. So if someone wants to use this for local and remote, both directory locations must be available.

Reply:
Yeah, there's really no way around that, unfortunately. We need a way to modify or use different configurations based on the plugin. Which was the original goal of this, but we're not there yet.